reporters/kamon-prometheus/src/main/resources/reference.conf

```diff
@@ -58,6 +58,23 @@ kamon.prometheus {
     }
   }
 
+  summaries: {
```

Comment:
Just a bit of nitpicking on this section and we are good to go:
  - Please, remove the `:` at the starting of each section. We don't use them anywhere else in our configurations and would prefer them to keep a consistent style.
  - Suggested text for the `metrics`: "Names of the histogram, timer and range sampler metrics that must be exported as summaries. All patterns are treared as Glob patterns."
  - For the quantiles section, please put the comment above the `quantiles` config.

Reply:
No problem. ^^
I agree with those points and changed them accordingly